hal/architecture/Linux/drivers/core/interrupt.cpp

```diff
@@ -239,13 +221,13 @@ void detachInterrupt(uint8_t gpioPin)
 		sysFds[gpioPin] = -1;
 	}
 
-	FILE *fp = fopen("/sys/class/gpio/unexport", "w");
-	if (fp == NULL) {
-		logError("Unable to unexport pin %d for interrupt\n", gpioPin);
-		exit(1);
-	}
-	fprintf(fp, "%d", gpioPin);
-	fclose(fp);
+//	FILE *fp = fopen("/sys/class/gpio/unexport", "w");
+//	if (fp == NULL) {
+//		logError("Unable to unexport pin %d for interrupt\n", gpioPin);
+//		exit(1);
+//	}
+//	fprintf(fp, "%d", gpioPin);
+//	fclose(fp);
```

Comment:
Would you mind deleting those lines instead of commenting them? We have git for history.